src/ValueObjects/Conflict.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Plank\Publisher\ValueObjects;
```

Comment:
Seeing this makes me wonder if we should just start including Spatie's Laravel-Data package as a dependency for all of our packages. Might be overkill, but honestly i wouldn't immediately discount it either

Reply:
I have also had this thought.